tracsubtickets/checker.py

```diff
@@ -34,8 +34,7 @@
 from api import NUMBERS_RE
 
 
-def check_subtickets(env):
-    db = env.get_db_cnx()
+def check_subtickets(db):
```

Comment:
I would pass `env` here instead and put a context manager around the queries `with env.db_query as db`, or even better, make a change like the following:

```
-    cursor.execute("SELECT parent, child FROM subtickets")
-    for row in cursor:
-         parent = int(row[0])
-        child = int(row[1])
+    for parent, child in env.db_query("""
+            SELECT parent, child FROM subtickets
+            """"):